analyzers/src/SonarAnalyzer.Common/Helpers/ProjectType.cs

```diff
@@ -25,6 +25,7 @@ namespace SonarAnalyzer.Helpers
     /// </summary>
     public enum ProjectType
     {
+        None,
```

Comment:
While reading the usage, I was little puzzled by the meaning of `None`. I suggest this to be clear about it.
```suggestion
        Unknown,
```